src/include/ipc4/base_fw.h

```diff
@@ -361,8 +361,10 @@ enum ipc4_fw_config_params {
 	IPC4_BUS_HARDWARE_ID = 25,
 	/* Total number of FW config parameters  */
 	IPC4_FW_CFG_PARAMS_COUNT,
+	/* FW context save on D3 entry */
+	IPC4_FW_CONTEXT_SAVE,
 	/* Max config parameter id */
-	IPC4_MAX_FW_CFG_PARAM = IPC4_FW_CFG_PARAMS_COUNT - 1,
+	IPC4_MAX_FW_CFG_PARAM = IPC4_FW_CONTEXT_SAVE - 1,
```

Comment:
@RanderWang @ujfalusi @mwasko @marcinszkudlinski to be on the safe side - can we add our new Intel reserved IDs as part of this PR and I think its going to be better if we also assign the numbers when updating this enum. i.e there is no ambiguos enum,  developers should always see
```
IPC4_SOME_ENUM = NUMBER,
```

Reply:
good idea!  I use 100, big enough for 10 years.

Reply:
> good idea! I use 100, big enough for 10 years.

be careful, our industry has a bad experinces with quotes like these ;)